README.md

```diff
@@ -67,6 +77,75 @@ See also [ERC721 — Non-Fungable Token Standard](https://github.com/ethereum/ei
 * https://www.nyse.com/regulation-a
 * https://www.sec.gov/files/Knyazeva_RegulationA%20.pdf
 * https://www.sec.gov/info/smallbus/secg/regulation-a-amendments-secg.shtml
+* https://legis.delaware.gov/json/BillDetail/GenerateHtmlDocument?legislationId=25730&legislationTypeId=1&docTypeId=2&legislationName=SB69
+
+## Crowdsale Details
+
+### Terms required by the SEC
+
+1. Terms (As defined by the SEC):
+
+    - Price per share
+
+        Priced in USD with a fixed ETH to USD conversion rate determined by the company at the start of the crowdsale.
+
+    - Nature of securities
+
+        Equity
+
+    - Closing date
+
+        See Crowdsale Parameters below.
+
+    - Amount of securities offered
+
+        See Crowdsale Parameters below.
+
+2. Implied Terms
+
+    - Statement as to the number of days left in the campaign
+
+        To be displayed on the Crowdsale web page, calculated based on current date vs crowdsale end date.
+
+    - Offering goal in dollars
+
+        To be displayed on the Crowdsale web page, calculated based on crowdsale cap and conversion rate.
+
+3. Non-Terms
+    - Minimum investment
+
+        Both to be displayed on the Crowdsale web page, as defined in the crowdsale contract in terms of minimum number of Tokens purchasable.
+
+    - Date when minimum is reached
+
+        To be displayed on the Crowdsale web page, based on data from the Crowdsale contract.
+
+    - Number of investors
+
+        To be displayed on the Crowdsale web page, based on data from the Token contract.
+
+    - Amount raised
+
+        To be displayed on the Crowdsale web page, based on data from the Token and Crowdsale contracts.
+        The Token contract will know the number of tokens issued `totalSupply()` and the crowdsale will know the USD to ETH conversion price.  When a Token is minted an event must be emitted that records the USD to ETH conversion price at the time of minting.
+
+    - The words “Closing Soon”
+
+        Will be displayed on the crowdsale website 7 days prior to closing.
+
+### Crowdsale Parameters
+
+* Maximum of 7,500 participants.
+* Overall cap is USD $2.5M (or equivalent)
```

Comment:
Perhaps we should put that its a Tier 1 Regulation A+ offering somewhere

Reply:
Also can we put the total number of shares we have as a hard constant? I forget if we decided 150k or is it 550k? @harrisonperl

Reply:
55000 shares on offer according to @harrisonperl

Reply:
@tavakyan we are aiming for Tier 2, but theoretically these tokens could be registered under either tier.

@tavakyan @davesag best to use variables for now, can't advertise details without appropriate disclaimers.